releasenotes/notes/0.19/fix-configurable-fake-backend-6a07ca5a6159baf5.yaml

```diff
@@ -2,11 +2,11 @@
 fixes:
   - |
     The member ``name`` of ``qiskit.test.mock.utils.ConfigurableFakeBackend``
-    has been changed to ``backend_name``. This was done to avoid a conflict with the
-    :meth:`~qiskit.providers.BackendV1.name` method inherited from the parent abstract
-    :class:`~qiskit.providers.BackendV1` class. This makes
+    has been changed to ``backend_name``. This was done to avoid a conflict with
+    the :meth:`~qiskit.providers.BackendV1.name` method inherited from the
+    parent abstract :class:`~qiskit.providers.BackendV1` class. This makes
     ``ConfigurableFakeBackend`` compatible with anything expecting a
     :class:`~qiskit.providers.BackendV1` object. However, if you were using the
-    ``name`` attribute directly before you will now need to either call it as a method or
-    access the ``backend_name`` attribute instead.
-    in several unittests because string is not callable.
+    ``name`` attribute directly before you will now need to either call it as a
+    method or access the ``backend_name`` attribute instead.  in several
+    unittests because string is not callable.
```

Comment:
```suggestion
    method or access the ``backend_name`` attribute instead.
```

Reply:
Dunno what happened here!  Fixed in 33ffd86f4.